app/templates/app/search.html

```diff
@@ -6,81 +6,161 @@
-                                </ul>
+                    <div class="col-md-3">
+                        <div class="form-group">
+                            <label for="id_subjects">Document Type</label>
+                            <div class="subjects-checkbox-container">
+                                {% for checkbox in filter.form.document_type %}
+                                    <div class="form-check">
+                                        {{ checkbox.tag }}
+                                        <label class="form-check-label" for="{{ checkbox.id_for_label }}">
+                                            {{ checkbox.choice_label }}
+                                        </label>
+                                    </div>
+                                {% endfor %}
                             </div>
-                        {% endfor %}
+                        </div>
                     </div>
-                    <div class="pagination">
-                        <span class="step-links">
-                            {% if documents.has_previous %}
-                                <a href="?page=1">&laquo; {% trans "first" %}</a>
-                                <a href="?page={{ documents.previous_page_number }}">{% trans "previous" }</a>
-                            {% endif %}
-                            <span class="current">
-                                Page {{ documents.number }} of {{ documents.paginator.num_pages }}.
-                            </span>
-                            {% if documents.has_next %}
-                                <a href="?page={{ documents.next_page_number }}">{% trans "next" %}</a>
-                                <a href="?page={{ documents.paginator.num_pages }}">{% trans "last" %} &raquo;</a>
-                            {% endif %}
-                        </span>
-                        <hr>
+                    <div class="col-md-3">
+                        <div class="form-group">
+                            <label for="id_subjects">Subjects</label>
+                            <div class="subjects-checkbox-container">
+                                {% for checkbox in filter.form.subjects %}
+                                    <div class="form-check">
+                                        {{ checkbox.tag }}
+                                        <label class="form-check-label" for="{{ checkbox.id_for_label }}">
+                                            {{ checkbox.choice_label }}
+                                        </label>
+                                    </div>
+                                {% endfor %}
+                            </div>
+                        </div>
                     </div>
+                    <div class="col-md-3">
+                        <div class="form-group">
+                            <label for="id_subjects">Languages</label>
+                            <div class="subjects-checkbox-container">
```

Comment:
Please consider a more general name for the container, as this is not about subjects.